From e9fc4f969bbcd01666af3817f227b4559b575c6c Mon Sep 17 00:00:00 2001
From: Susmitha Gummalla <vgummall@codeaurora.org>
Date: Wed, 24 Jun 2015 22:38:25 -0700
Subject: [PATCH 23/25] Camera: Add feature extensions

This change includes below commits:

    Camera bringup changes system-core
    Change-Id: I1cf98641eca9096bd27645e07ea802646ea1fb96

    system/core: Fix for HAL compilation issues while integrating HAL 1.0
    Change-Id: Iead9c1ade279b64c5cbdf4d2de1a8b695939c52a

    Camera: Add enum to specify the frame type
    Added enum to specify the frame type of either fd/data buffer
    CRs-fixed: 654901
    Change-Id: I1c0b1a2c6a1425cdb6650cdfc20ca65835a1b81f

Change-Id: I654a40661e6e101da2a06986abeceb20639cccd9

Conflicts:
	libsystem/include/system/camera.h
---
 libsystem/include/system/camera.h | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/libsystem/include/system/camera.h b/libsystem/include/system/camera.h
index 264d7100e4..10acfe6374 100644
--- a/libsystem/include/system/camera.h
+++ b/libsystem/include/system/camera.h
@@ -200,7 +200,6 @@ enum {
      * IMPLEMENTATION_DEFINED, then HALv3 devices will use gralloc usage flags
      * of SW_READ_OFTEN.
      */
-#ifndef CAMERA_VENDOR_L_COMPAT
     CAMERA_CMD_SET_VIDEO_FORMAT = 11,
 
     CAMERA_CMD_VENDOR_START = 20,
@@ -220,20 +219,6 @@ enum {
     CAMERA_CMD_METADATA_ON = CAMERA_CMD_VENDOR_START + 6,
     CAMERA_CMD_METADATA_OFF = CAMERA_CMD_VENDOR_START + 7,
     CAMERA_CMD_VENDOR_END = 200,
-#else
-    /**
-     * Values used by older HALs, provided as an option for compatibility
-     */
-    CAMERA_CMD_HISTOGRAM_ON = 11,
-    CAMERA_CMD_HISTOGRAM_OFF = 12,
-    CAMERA_CMD_HISTOGRAM_SEND_DATA = 13,
-    CAMERA_CMD_LONGSHOT_ON = 14,
-    CAMERA_CMD_LONGSHOT_OFF = 15,
-    CAMERA_CMD_STOP_LONGSHOT = 16,
-    CAMERA_CMD_METADATA_ON = 100,
-    CAMERA_CMD_METADATA_OFF = 101,
-    CAMERA_CMD_SET_VIDEO_FORMAT = 102,
-#endif
 };
 
 /** camera fatal errors */
-- 
2.17.1

